src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Square implements Figure {
```

Comment:
The class `Square` should not start with an empty line. Please remove the unnecessary empty line at the beginning of the class definition.